### PR TITLE
some announcement qol

### PR DESCRIPTION
**Pull request by Neocloudy**
## About The Pull Request
- Orbital bombardment now plays screentext for every human, and changed the content of the message because "get out of danger close" is dumb
  - If possible, I want to make this play over other sources of screentext (Send Orders especially, as it is used almost always before or during an OB) at some point, as a simple narration for the type of an incoming OB is *definitely* more important than whatever the PVT who just unlocked FC has to say. Maybe for a future PR.
- "Send Orders" will now clarify the faction of the sender (EDIT: regardless of TGMC affiliation)
  - `faction_to_acronym` global list for getting faction acronyms to make this possible.

image
image
image
image
## Why It's Good For The Game
- Orbital bombardment playing screentext is a good idea to try and get attention of marines who can't afford to look in chat in the middle of combat
- Being able to tell what faction an announcement is from is good. Also, future coders can use the `faction_to_acronym` global list for something interesting
## Changelog
:cl:
qol: Orbital bombardments now play screentext to humans that includes the type of the OB
qol: Send Orders will now include the sender's faction acronym and job
spellcheck: The content of the orbital bombardment announcement is now "Evacuate the impact zone immediately!" instead of "Get out of danger close!"
code: "faction_to_acronym" global list for getting faction acronyms
/:cl:

### CODE REVIEW

**Comment by Lumipharon** [comment]
why would you use the full name instead of TGMC/SOM/ICC

File: code/modules/screen_alert/command_alert.dm [hunk 1]
```diff
@@ -77,9 +77,12 @@
 		return
 	for(var/mob/faction_receiver in alert_receivers)
 		if(faction_receiver.faction == human_owner.faction || isdead(faction_receiver))
-			faction_receiver.play_screen_text("<span class='maptext' style=font-size:24pt;text-align:center valign='top'><u>COMMAND ANNOUNCEMENT:</u></span><br>" + text, /atom/movable/screen/text/screen_text/command_order)
+			var/faction_title = "Command" // the default title, for tgmc announcements
+			if(human_owner.faction != FACTION_TERRAGOV)
+				faction_title = "[GLOB.faction_to_acronym[human_owner.faction]] Command" // if the sender doesn't belong to the TGMC, clarify the faction in the title
```
Comment (Lumipharon) [inline code/modules/screen_alert/command_alert.dm:82]: I wouldn't bother with the exclusion for tgmc tbh.

Comment (Lumipharon) [inline code/modules/screen_alert/command_alert.dm:82]: Should probs not assume the faction will always be in the list (even if it should)

**Review by Lumipharon — CHANGES_REQUESTED** [review]

File: code/modules/screen_alert/command_alert.dm (hunk 1 above)
Comment (Neocloudy) [reply code/modules/screen_alert/command_alert.dm:82]: how should I implement this? I am basically clueless as to how I would make it check if the leader's faction is in that list or something

Comment (Neocloudy) [reply code/modules/screen_alert/command_alert.dm:82]: TGMC exclusion is an antique from when I worked on 14957, I want send orders from TGMC leaders to look like a CIC announcement. I'll change this if you really want though

Comment (Lumipharon) [reply code/modules/screen_alert/command_alert.dm:82]: ternary or something

Comment (Lumipharon) [reply code/modules/screen_alert/command_alert.dm]: its just consistant

**Review by Lumipharon — APPROVED** [review]

**Pull request merged**